Notify/WEB/ClientApp/src/components/Authentication/SelectorLogReg.js

```diff
@@ -0,0 +1,18 @@
+﻿import React from 'react';
+
+const SelectorLogReg = ({ selectLogReg }) => {
```

Comment:
okay, i've finally understood logic of this rogreg component.

you should use react router to implement such logic. define two routes: /login, /register and use your LoginForm and RegisterFrom components.
and in AppHeader you can just use 
`<Link to='/login' />`
and
`<Link to='/register' />`